### PR TITLE
Disable "Terminate" button when there's unprocessed signals

### PR DESCRIPTION
**Pull request by eagletmt**
To prevent multiple clicks on "Terminate" button.

@cookpad/dev-infra please review

### CODE REVIEW

File: app/controllers/kuroko2/executions_controller.rb [hunk 1]
```diff
@@ -7,7 +7,7 @@ def index
   end
 
   def destroy
-    if @execution.try!(:pid)
+    if @execution.terminatable?
```
Comment (ganmacs) [inline app/controllers/kuroko2/executions_controller.rb]: Checking terminatable(L10) and creating an execution record(L13) are not atomic operations. Is it acceptable?

Comment (eagletmt) [reply app/controllers/kuroko2/executions_controller.rb]: added a unique index to prevent duplication

**Comment by eagletmt** [comment]
@cookpad/infra please review

**Comment by itkq** [comment]
@eagletmt I'm going to merge https://github.com/cookpad/kuroko2/pull/128 and release it as v0.5.2. This PR seems merge-ready, so do you think it is good to merge and ship this at the same time?

**Pull request closed without merging**